queries.sql

```diff
@@ -19,4 +21,56 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE name NOT LIKE 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- query and update animals table -------------------------------------
+
+ALTER TABLE animals ADD COLUMN species varchar(50);
+
+BEGIN; -- starts transaction
+
+UPDATE animals SET species = 'unspecified';
+
+ROLLBACK; -- ends transaction
+
+BEGIN; -- starts transaction
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE species = '';
+
+COMMIT; -- ends transaction
+
+BEGIN; -- starts transaction
+
+DELETE FROM animals;
+
+SELECT count(id) FROM animals;
```

Comment:
As the previous reviewer mentioned, it's important that the verification is done AFTER the ROLLBACK 🤓 Please be sure to make this change so that your project meets the requirements 🔥